test/base64-option.test.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable
+  prefer-destructuring,
+*/
+import webpack from '@webpack-contrib/test-utils';
+
+describe('base64 option', () => {
+  test('{undefined}', async () => {
+    const config = {
+      loader: {
+        test: /\.png$/,
+        options: {},
+      },
+    };
+
+    const stats = await webpack('fixture.js', config);
+    const { source } = stats.toJson().modules[0];
+
+    expect(source).toMatchSnapshot();
+  });
+
+  test('{Boolean} false', async () => {
+    const config = {
+      loader: {
+        test: /\.png$/,
```

Comment:
Please test with an actual `.svg` file here. Create a `base64` folder in `test/fixtures` with the following contents (you can copy `file.svg` => `base64/file.svg`)

```
fixtures
|– base64
||– file.svg
||– fixture.js // import svg from './file.svg' ...
|– ...
|–fixture.js 
```

and 

```diff
-  const stats = await webpack('fixture.js', config);
+  const stats = await webpack('base64/fixtures.js', config);
```